tests/e2e/support/commands.js

```diff
@@ -881,13 +879,8 @@ Cypress.Commands.add('playJackFromSevenOpponent', (card, target) => {
       }
 
       const cardId = foundCard.id;
-      let targetId;
+      const targetId = foundTarget.id;
 
-      if (target !== -1) {
-        targetId = foundTarget.id;
-      } else {
-        targetId = -1;
-      }
```

Comment:
Moved `cy. sevenDiscardOpponent()` into it's own command and simplified this command (`cy. playJackFromSevenOpponent()` so it's only for actually playing the jack